docs/rebuild_admin.rst

```diff
@@ -0,0 +1,436 @@
+appropriate email address for ``alerts@example.com``):
+
+.. code:: sh
+
+ ssh mon sudo gpg --homedir=/var/ossec/.gnupg --export --armor alerts@example.com > ossec.pub
+ gpg --import ossec.pub
+ 
+You will be prompted for the fingerprints for both keys during the next step. To view their fingerprints, use the command:
+
+.. code:: sh
+ 
+ gpg -k --fingerprint
+
+Retrieve OSSEC alert configuration details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In addition to the OSSEC alert GPG key you retrieved above, you'll also need 
+the following configuration information:
+
+ - SMTP server
+ - SMTP port
+ - SASL username
+ - SASL domain
+ - SASL password
+
+To retrieve these values, use the following command in the terminal:
+
+.. code:: sh
+
+ ssh mon sudo cat /etc/postfix/sasl_passwd
+
+This will return a line like:
+
+.. code:: sh
+ 
+ [smtp.gmail.com]:587 testossec@gmail.com:AwfulPassword
+
+In this example, ``smtp.gmail.com`` is the SMTP server, ``587`` is the SMTP port,
+``testossec`` is the SASL username, ``gmail.com`` is the SASL domain, and 
+``AwfulPassword`` is the SASL password.
+
+(Optional) Retrieve HTTPS certificate files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If your *Source Interface* was configured to use HTTPS, you will need to copy
+three related files from the *Application Server* to the *Admin Workstation*.
+
+To retrieve these files, use the commands:
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop/install_files/ansible-base
+   ssh app sudo tar -c -C /var/lib ssl/  | tar xvf - 
+
+These commands will create a directory named 
+``~/Persistent/securedrop/install_files/ansible-base/ssl``
+on the *Admin Workstation*, containing your instance's SSL certificate, 
+certificate key, and chain file. When prompted for the names of these files
+during the next step, you should specify them relative to the 
+``install_files/ansible-base`` directory, i.e. as ``ssl/mydomain.crt``.
+
+Step 5: Configure and back up the application
+=============================================
+
+Next, configure the application using the files and info retrieved in the 
+previous steps. To do so, connect to the Tor network on the
+*Admin Workstation*, open a Terminal and run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin setup
+ ./securedrop-admin sdconfig
+
+.. note:: The ``./securedrop-admin setup`` command may take several minutes to complete, and may
+ fail due to network issues. If it fails, it's safe to run again.
+
+When prompted, fill in the values retrieved in the previous steps.
+
+Once complete, configure the *Admin Workstation* using the command:
+
+.. code:: sh
+
+  ./securedrop-admin tailsconfig
+
+This will set up desktop shortcuts for the *Source* and *Journalist Interfaces*,
+and configure adminstrative access to the servers.
+
+Once the ``./securedrop-admin tailsconfig`` command is complete, verify that you can connect to 
```

Comment:
Is there a reason why tailsconfig is done prior to install? the overview states install before tailsconfig https://github.com/freedomofpress/securedrop/pull/4522/files#diff-5aa6154ded0c00ed8eb19ab7a2b2031eR24

Since the ssh config is valid, maybe you want the user to confirm at this step that the desktop shortcuts for source and journalist interface are working?

Reply:
Yup, that'd be prudent. Updated.